src/Compilers/CSharp/Test/Emit3/RefReadonlyParameterTests.cs

```diff
@@ -774,9 +774,6 @@ public static void M(ref readonly int x) { }
             // (7,25): error CS1676: Parameter 1 must be declared with the 'ref readonly' keyword
             // Expression<D> e4 = (int p) => C.M(in p);
             Diagnostic(ErrorCode.ERR_BadParamRef, "p").WithArguments("1", "ref readonly").WithLocation(7, 25),
-            // (7,28): error CS1661: Cannot convert lambda expression to type 'Expression<D>' because the parameter types do not match the delegate parameter types
-            // Expression<D> e4 = (int p) => C.M(in p);
```

Comment:
It looks like we're no longer reporting a "conversion error" for this case, we're just reporting an error for ref kind mismatch, even though the lambda is not convertible. Compare with the parameter type mismatch where we report a conversion error as well.
```csharp
D d4A = (int p) => { };               // CS1676 only
D d4B = (ref readonly byte p) => { }; // CS1661, CS1678

delegate void D(ref readonly int p);
```

Reply:
Yes.  I think that's fine.  The errors happen in phases, and telling someone they got the ref-ness wrong first is fine.  They can fix that, and then find out they got the type part wrong as well.